service/history/transferQueueActiveTaskExecutor_test.go

```diff
@@ -1279,18 +1279,20 @@ func (s *transferQueueActiveTaskExecutorSuite) TestProcessCloseExecution_DeleteA
 			execution.GetWorkflowId(),
 			execution.GetRunId(),
 		),
-		Version:             s.version + 1,
+		Version:             s.version,
```

Comment:
why changing this test?

Reply:
This test is not correct. Before it return silently because the version is mismatch. After we return an error, this test failed unexpected. So it needs the same version to test it logic.